readme.md

```diff
@@ -44,12 +44,23 @@ Called during the handling of a Micropub request. The content generation functio
 
 `micropub_syndicate-to( $synd_urls, $user_id )`
 
-Called to generate the list of `syndicate-to` targets to return in response to a query. Returns `$synd_urls`, an array, possibly modified.
-
+Called to generate the list of `syndicate-to` targets to return in response to a query. Returns `$synd_urls`, an array, possibly modified. This filter is empty by default
 `micropub_query( $resp, $input )`
 
 $resp defaults to null. If the return value is non-null, it should be an associative array that is encoded as JSON and will be returned in place of the normal micropub response.
 
+`disable_micropub_auth( $boolean )`
```

Comment:
mind mentioning explicitly what the arg and return value types are for these new filters?